Dockerfile.cli.jdk8

```diff
@@ -24,7 +24,7 @@ ADD --chown=dspace . /app/
 COPY dspace/src/main/docker/local.cfg /app/local.cfg
 
 # Build DSpace.  Copy the dspace-install directory to /install.  Clean up the build to keep the docker image small
-RUN mvn package -Dmirage2.on=true && \
+RUN mvn package -P dspace-api -Dmirage2.on=true && \
```

Comment:
Sorry, I didn't submit my comment properly.
We have to exclude, like:
`RUN mvn package -P'!dspace-solr,!dspace-jspui,!dspace-xmlui,!dspace-rest,!dspace-xmlui-mirage2,!dspace-rdf,!dspace-sword,!dspace-swordv2,!dspace-oai' && \`